tests/integration.test.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path');
+
+const webdriver = require('selenium-webdriver');
+const { Context } = require('selenium-webdriver/firefox');
+
+// The geckodriver package downloads and installs geckodriver for us.  We use
+// it by requiring it.
+require('geckodriver');
+
+const { mozCompare } = require('../src');
+const { ALL_VERSIONS } = require('./helpers');
+
+describe(__filename, () => {
+  jest.setTimeout(20000);
+
+  let driver;
+
+  beforeAll(() => {
+    driver = new webdriver.Builder().forBrowser('firefox').build();
+    driver.setContext(Context.CHROME);
+  });
+
+  afterAll(() => {
+    return driver.quit();
+  });
+
+  it('should have versions to test', () => {
+    expect(ALL_VERSIONS).not.toHaveLength(0);
+  });
+
+  it.each(ALL_VERSIONS)(
+    'matches Firefox Services.vc.compare("%s", "%s")',
+    async (a, b) => {
+      let fxVcCompare = await driver.executeScript(
+        `return Services.vc.compare("${a}", "${b}");`
+      );
+
+      if (fxVcCompare < 0) {
+        fxVcCompare = -1;
+      } else if (fxVcCompare > 0) {
+        fxVcCompare = 1;
+      }
```

Comment:
eh, I was curious about why was this needed and so I ended up executing the test locally 

and I learned that `Services.vc.compare('1.1pre1aa', '1.1pre1a');` is returning 97 and the opposite -97 :-D
that kind of surprised me (I'm guessing that 97 is due to the fact that 'a' char code is 97, which seems to be confirmed also by the fact that if I replace "1.1pre1aa" with "1.1pre1ab" it returns 98 instead).

Well, it was more like a curiosity, in the end we only care about the sign and so this is fine.